internal/git/git.go

```diff
@@ -108,6 +110,14 @@ func (r *Repo) TrunkBranches() ([]string, error) {
 	return branches, nil
 }
 
+func (r *Repo) GetRemoteName() string {
+	if config.Av.Remote != "" {
```

Comment:
Which is better, setting it as the initial value (at [here](https://github.com/aviator-co/av/pull/340/files#diff-54c7c1af5fa8d5db4dc49f0e8e80e93ba2b1183ba4d5c9e2e5729e6deae6a3cdR71)) or checking it here?

Reply:
This is fine.